### PR TITLE
Replace repeating requires browser restart text with * in about:settings

### LINKED ISSUE #7342: Replace repeating replace browser restart text with *
Replace repeating `requires browser restart` text with `*`  in about:settings

### PR DESCRIPTION
**Pull request by srirambv**
- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).

Test Plan:
All options which requires restart should contain * next to it and at the bottom of the page should contain `* Requires browser restart`

fix #7342

Auditors: @bradleyrichter @cezaraugusto

### CODE REVIEW

**Comment by luixxiul** [comment]
Changes https://github.com/brave/browser-laptop/pull/7343/files#diff-e3eeb751016b2ce9f8278efce585a461 should be reverted

**Comment by srirambv** [comment]
changes reverted for preferences.js

File: app/extensions/brave/locales/en-US/preferences.properties [hunk 1]
```diff
@@ -333,6 +333,7 @@ defaultBrowser=Brave is your default browser.
 notDefaultBrowser=Brave is not your default browser.
 setAsDefault=Set as default…
 checkDefaultOnStartup=Always check on startup
-useTorrentViewer=Enable Torrent Viewer (requires browser restart)
+useTorrentViewer=Enable Torrent Viewer *
 dashboardSettingsTitle=Dashboard
 dashboardShowImages=Show images
+requireRestart=* Requires browser restart
```
Comment (cezaraugusto) [inline app/extensions/brave/locales/en-US/preferences.properties]: I think there's a typo here (which breaks all l10n for that). 

Should be `requiresRestart` instead

Comment (srirambv) [inline app/extensions/brave/locales/en-US/preferences.properties]: Fixed typo

**Pull request merged**